### ISSUE TITLE
base-lcov-file is not required

### ISSUE THREAD

**Issue by gabsmprocha** [barecheck/code-coverage-action#148]
As mentioned in the readme the base-lcov-file parameter is not mandatory, but if I don't inform the action it fails

**Comment by vitaliimelnychuk** [comment]
Hello @gabsmprocha!

Thanks for reaching out. Just to clarify the issue, If you don't pass `base-lcov-file` the action fails?

**Comment by gabsmprocha** [comment]
> Hello @gabsmprocha!
> 
> Thanks for reaching out. Just to clarify the issue, If you don't pass `base-lcov-file` the action fails?
> 
> if yes, you're right the action should just skip comparison with the base file part when the base file is not passed.

yes, it fails

**Comment by wkjung** [comment]
@vitaliimelnychuk It seems like the value is initialized even if I don't pass the value.
When I don't pass github action claims like
```
Run barecheck/code-coverage-action@v0.5.1
lcov-file: ./build/ccov/coverage.lcov
base-lcov-file: ./coverage/lcov.info
(node:1410432) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open './coverage/lcov.info'
    at Object.openSync (fs.js:462:3)
    at Object.readFileSync (fs.js:364:35)
```

My yml:
```
    - name: Generate Code Coverage report
      id: code-coverage
      uses: barecheck/code-coverage-action@v0.5.1
      with:
        barecheck-github-app-token: <...>
        lcov-file: "./build/ccov/coverage.lcov"
        send-summary-comment: true
        show-annotations: "warning" # Possible options warning|error
```

**Comment by vitaliimelnychuk** [comment]
Hello @wkjung @gabsmprocha!
You're right something wrong with the check for the property. I agree we should skip the comparison with the base file once the file is not sent. 

A quick fix would be just to pass `base-lcov-file: false`. 

I will take a look or if you want you also can do this. We need to just have another check here https://github.com/barecheck/code-coverage-action/blob/master/src/index.js#L29.

Let me know if you need any help meanwhile.

**Comment by wkjung** [comment]
@vitaliimelnychuk Thanks for the quick response.

Unfortunately, the quickfix `base-lcov-file: false` does not seem to work.
It outputs like:

```
Run barecheck/code-coverage-action@v0.5.1
lcov-file: ./build/ccov/coverage.lcov
base-lcov-file: false
(node:1493480) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'false'
    at Object.openSync (fs.js:462:3)
    at Object.readFileSync (fs.js:364:35)
```

with
```
        lcov-file: "./build/ccov/coverage.lcov"
        send-summary-comment: true
        show-annotations: "warning" # Possible options warning|error
        base-lcov-file: false
```

**Comment by vitaliimelnychuk** [comment]
@vitaliimelnychuk @gabsmprocha Just released a fix in [`0.6.1`](https://github.com/barecheck/code-coverage-action/pull/152) version. Please try it out and let me know if everything is fine
